tests_e2e/orchestrator/lib/agent_test_suite.py

```diff
@@ -522,9 +521,9 @@ def _execute_test_suite(self, suite: TestSuiteInfo) -> bool:
 
                         test_success: bool = True
 
+                        load_test = test.test_class(self.context)
```

Comment:
btw, could we rename this variable "load_test" doesn't have a lot of meaning, I think. Maybe "test_instance"?